docs/manual-qa/hf_esg6_tutorial_test.mdx

```diff
@@ -0,0 +1,29 @@
+## Testing: Hugging Face - ESG (6) Tutorial
+
+Testing CREATE MODEL
+```sql
+CREATE MODEL mindsdb.hf_esg_6
+PREDICT PRED
+USING
+engine = 'huggingface',
+task = 'text-classification',
+model_name = 'yiyanghkust/finbert-esg',
+input_column = 'text';
+
+```
+
+```sql
+[export.md](https://github.com/MahenderPoshaboina/mindsdb/files/11634666/export.md)
```

Comment:
Instead of linking an `.md` file, please copy and paste its content here. (This linking could be used if you provided screenshots)